src/test/java/com/game/Battleship/controllers/BoardControllerTests.java

```diff
@@ -0,0 +1,44 @@
+package com.game.Battleship.controllers;
+import com.game.Battleship.models.Board;
+import com.game.Battleship.models.Cell;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import  org.springframework.boot.test.context.SpringBootTest;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+
+public class BoardControllerTests {
+    @Autowired
+    private TestRestTemplate template;
+
+    @Test
+    public void boardEndpointContainsSize(){
+
+        Board response = template.getForObject("/board", Board.class);
+        Assertions.assertEquals(10, response.getSize());
+
+    }
+
+    @Test
+    public void boardFieldSize(){
```

Comment:
Can you think of a better test name for this? This test name doesn't convey a whole lot and may force readers (ie. other developers on the project) to have to look into the actual test code to see what's going on. Code should read like a story 😉

Reply:
Done